guides/common/modules/ref_system-requirements.adoc

```diff
@@ -3,37 +3,76 @@
 [id="system-requirements_{context}"]
 = System requirements
 
-The following requirements apply to the networked base operating system:
-
-* x86_64 architecture
 ifdef::satellite[]
-* The latest version of {RHEL} 9
+{ProductName} is fully supported on both physical systems and virtual machines that run on hypervisors that are supported to run {RHEL}.
+For more information about certified hypervisors, see https://access.redhat.com/articles/973163[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization, Red Hat OpenShift Virtualization and Red Hat Enterprise Linux with KVM].
+
 endif::[]
-* 4-core 2.0 GHz CPU at a minimum
+Follow these system requirements when installing {ProductName}:
+
+* Install {ProductName} on a freshly provisioned system that servers no other function except to run {ProductName}.
+Do not use an existing system because the {Project} installer will affect the configuration of several components.
+* Ensure you have administrative user (root) access to the system.
+* Ensure the system meets the following requirements:
+** 4-core 2.0 GHz CPU or higher
 
 ifeval::["{context}" == "{project-context}"]
-* A minimum of {project-minimum-memory} RAM is required for {ProjectServer} to function.
+** {project-minimum-memory} RAM or higher
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+** 12 GB RAM or higher
+endif::[]
 ifdef::katello,satellite[]
-In addition, a minimum of 4 GB RAM of swap space is also recommended.
+** 4 GB RAM of swap space or higher
 endif::[]
-{Project} running with less RAM than the minimum value might not operate correctly.
+
+ifdef::katello,satellite[]
+** A unique host name, which can contain lower-case letters, numbers, dots (.) and hyphens (-)
 endif::[]
 
-ifeval::["{context}" == "{smart-proxy-context}"]
-* A minimum of 12 GB RAM is required for {SmartProxyServer} to function.
-In addition, a minimum of 4 GB RAM of swap space is also recommended.
-{SmartProxy} running with less RAM than the minimum value might not operate correctly.
+* Ensure that the system does not have the following users provided by external identity providers.
+These user accounts can cause conflicts with the local users that {ProductName} creates:
+** {apache-user}
+ifeval::["{context}" == "{project-context}"]
+** foreman
+endif::[]
+** foreman-proxy
+** postgres
+ifdef::katello,satellite,orcharhino[]
+** pulp
+endif::[]
+** puppet
+** redis
+ifdef::katello,satellite,orcharhino[]
+ifeval::["{context}" == "{project-context}"]
+** tomcat
+endif::[]
 endif::[]
 
-ifdef::katello,satellite[]
-* A unique host name, which can contain lower-case letters, numbers, dots (.) and hyphens (-)
+* If you use custom certificates, ensure that the Common Name (CN) of the custom certificate is a fully qualified domain name (FQDN).
+Ensure that {ProjectServer} and {SmartProxyServer} do not support shortnames in the hostnames.
```

Comment:
The wording is off in my eyes. Either "X does not support Y" or "Ensure that X does positive thing Y".